### ISSUE TITLE
Images on /marketing/business-tools are missing alt text

### ISSUE THREAD

**Issue by youbrokesomething** [Automattic/wp-calypso#75602]
### Quick summary

Seven images on /marketing/business-tools are missing alt text. If images aren't important to the context of the page, they should have an empty alt-text tag.

[You can use this guide to determine if an alt-tag should be empty or have text.](https://www.w3.org/WAI/tutorials/images/decision-tree/#:~:text=wasn't%20there%3F-,Yes%3A,the%20link%20or%20action%20taken)

### Steps to reproduce

1. On /marketing/business-tools
2. Inspect images on the page or use an accessibility checker like axe DevTools Chrome extension.

### What you expected to happen

All images should have an alt tag even if text isn't needed.

### What actually happened

Seven images are missing an alt tag.

### Impact

Some (< 50%)

### Available workarounds?

No but the platform is still usable

**Comment by danielbachhuber** [comment]
@worldomonation Given you've self-assigned, I'll remove this one from our board.

**Comment by worldomonation** [comment]
> @worldomonation Given you've self-assigned, I'll remove this one from our board.

Oh, sorry about that!
